### PR TITLE
Feature/add category boxes to contact form

### PR DESCRIPTION
**Pull request by bwreDev**

### CODE REVIEW

File: web/pages/api/contact.js [hunk 1]
```diff
@@ -24,11 +24,13 @@ export default async function handler(req, res) {
         `<p>Name: ${req.body.name}</p>
         <p>Email: ${req.body.emailAddress}</p>
         <p>Phone: ${req.body.phoneNumber}</p>
-        <p>Message: ${req.body.message}</p>`,
+        <p>Message: ${req.body.message}</p>
+        <p>Preference: ${req.body.preference}</p>`,
```
Comment (michaelbonner) [inline web/pages/api/contact.js:28]: @bwreDev We should probably serialize this a little better. Probably just a join with commas would be fine for now

File: web/pages/api/contact.js [hunk 2]
```diff
@@ -24,11 +24,13 @@ export default async function handler(req, res) {
...
         `Name: ${req.body.name}\n
         Email: ${req.body.emailAddress}\n
         Phone: ${req.body.phoneNumber}\n
-        Message: ${req.body.message}`
+        Message: ${req.body.message}\n
+        Preference: ${req.body.preference}`
```
Comment (michaelbonner) [inline web/pages/api/contact.js:33]: Same here as above

File: web/tailwind.config.js [hunk 3]
```diff
@@ -36,5 +36,6 @@ module.exports = {
   plugins: [
     require('@tailwindcss/typography'),
     require('@tailwindcss/aspect-ratio'),
+    require('@tailwindcss/forms'),
```
Comment (michaelbonner) [inline web/tailwind.config.js:39]: Good call on tailwind forms

**Review by michaelbonner — CHANGES_REQUESTED** [review]

File: web/pages/api/contact.js (hunk 2 above)
Comment (bwreDev) [reply web/pages/api/contact.js]: I added a `.join(', ') ` to each of these.

**Review by michaelbonner — APPROVED** [review]

**Review by michaelbonner — APPROVED** [review]

**Review by michaelbonner — APPROVED** [review]

**Pull request merged**